x-pack/plugins/secops/public/containers/overview/overview_host/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getOr } from 'lodash/fp';
+import React from 'react';
+import { Query } from 'react-apollo';
+import { pure } from 'recompose';
+
+import { GetSiemOverviewHostQuery, SiemOverviewHostData } from '../../../graphql/types';
+import { createFilter } from '../../helpers';
+import { QueryTemplateProps } from '../../query_template';
+
+import { overviewHostQuery } from './index.gql_query';
+
+export interface SiemOverviewHostArgs {
+  id: string;
+  overviewHost: SiemOverviewHostData;
+}
+
+export interface SiemOverviewHostProps extends QueryTemplateProps {
+  children: (args: SiemOverviewHostArgs) => React.ReactNode;
+  sourceId: string;
+  endDate: number;
+  poll: number;
+  startDate: number;
+}
+
+export const SiemOverviewHostQuery = pure<SiemOverviewHostProps>(
+  ({ id = 'overviewHostQuery', children, filterQuery, sourceId, startDate, endDate }) => (
+    <Query<GetSiemOverviewHostQuery.Query, GetSiemOverviewHostQuery.Variables>
+      query={overviewHostQuery}
+      fetchPolicy="cache-and-network"
+      variables={{
+        sourceId,
+        timerange: {
+          interval: '12h',
+          from: startDate!,
+          to: endDate!,
+        },
+        filterQuery: createFilter(filterQuery),
+      }}
+    >
+      {({ data }) => {
```

Comment:
Question: Should we return loading state so we can show to the end user that we are loading the data for them.

Reply:
Design says to return '--' here for loading: https://elastic.slack.com/archives/C7QC1JV6F/p1554227120056100